### PR TITLE
Fix TestFlight pipeline

### PR DESCRIPTION
**Pull request by testableapple**
### 🎯 Goal

Fix TestFlight pipeline

### 📝 Summary

There were multiple issues in `Info.plist` files in all demo targets, which are fixed now.

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] This change follows zero ⚠️ policy (required)
- [ ] This change should receive manual QA
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (Docusaurus, tutorial, CMS)

### 🎁 Meme

giphy-downsized-large (1)

### CODE REVIEW

File: StreamVideo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3289,9 +3289,10 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 15;
+				CURRENT_PROJECT_VERSION = 1;
```
Comment (martinmitrevski) [inline StreamVideo.xcodeproj/project.pbxproj:3292]: why is this moved back to 1?

File: StreamVideo.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3289,9 +3289,10 @@
...
 				DEVELOPMENT_TEAM = EHV7XZLAHA;
-				GENERATE_INFOPLIST_FILE = YES;
+				GENERATE_INFOPLIST_FILE = NO;
+				GENERATE_PKGINFO_FILE = YES;
```
Comment (martinmitrevski) [inline StreamVideo.xcodeproj/project.pbxproj:3295]: what's this and why do we need to change it?

**Review by martinmitrevski — COMMENTED** [review]
Looks good 👍  Added few small comments. Also, would be good to test this with a fresh install on a device, to make sure the demo app shows the correct permissions the first time.

File: StreamVideo.xcodeproj/project.pbxproj (hunk 1 above)
Comment (testableapple) [reply StreamVideo.xcodeproj/project.pbxproj:3292]: It always should be one as it's fully managed by fastlane. Changing it manually will lead to issues in the TestFlight build.

File: StreamVideo.xcodeproj/project.pbxproj (hunk 2 above)
Comment (testableapple) [reply StreamVideo.xcodeproj/project.pbxproj:3295]: I transfered these options from Chat repo. Without them the generated `.ipa` file used the wrong `Info.plist` which was breaking the TestFlight uploading process.

**Comment by testableapple** [comment]
@martinmitrevski, I checked the fresh install, `notifications`, `microphone` and `camera` alerts popped up as expected 👍

**Review by martinmitrevski — APPROVED** [review]
